echidna/settings.py

```diff
@@ -1,3 +1,5 @@
+import echidna
+
```

Comment:
Just a quick thing, can you move the import to after the module docstring, because sphinx doesn't print it if it's not right at the top of the file. Cheers